components/raftstore/src/store/snap.rs

```diff
@@ -1886,6 +1891,45 @@ impl SnapManagerBuilder {
     }
 }
 
+/// `TabletSnapManager` manager tablet snapshot and shared between raftstore v2.
+/// It's similar `SnapManager`, but simpler in tablet version.
+///
+///  TODO:
+///     - add Limiter to control send/recv speed
+///     - clean up expired tablet checkpointer
+#[derive(Clone)]
+pub struct TabletSnapManager {
+    // directory to store snapfile.
+    base: String,
+}
+
+impl TabletSnapManager {
+    pub fn new<T: Into<String>>(path: T) -> Self {
+        Self { base: path.into() }
+    }
+
+    pub fn init(&self) -> io::Result<()> {
```

Comment:
Why not do this immediately in `new`?

Reply:
will add more logic such as clean the stale `checkpointer` in here，same as v1.